scripts/style_drift_scan.py

```
@@ -0,0 +1,371 @@
+        yp = library_root / category / "_index.yaml"
+        if not yp.exists():
+            continue
+        try:
+            import yaml
+            data = yaml.safe_load(yp.read_text()) or {}
+            for entry in data.get("elements", []):
+                sid = entry.get("style_id")
+                if sid:
+                    style_ids.add(sid)
+        except Exception:  # noqa: BLE001
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)